ios/Samples/WesterosEmployees-hybrid/App/WesterosEmployees-hybrid/Resources/js/blog.js

```diff
@@ -0,0 +1,47 @@
+function modifyItem() {
+    
+    var allLinks = document.getElementsByTagName('a');
+    var allImages = document.getElementsByTagName('img');
+    
+    for (var i = 0; i < allLinks.length; i++) {
+        allLinks[i].removeAttribute('href');
```

Comment:
It doesn't matter much. But onclick is a bit more explicit, you have to know less JS and html to know how it works than href. I wouldn't change it right now, I questioned to see if there was another motive.